examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -159,7 +163,7 @@ class ClusterCommand : public InteractionModelCommands, public ModelCommand, pub
         if ((path.mEndpointId == chip::kRootEndpointId) && (path.mClusterId == chip::app::Clusters::IcdManagement::Id) &&
             (path.mCommandId == chip::app::Clusters::IcdManagement::Commands::UnregisterClient::Id))
         {
-            ClearICDEntry(mScopedNodeId);
+            ClearICDEntry(mPeerNodeId);
```

Comment:
OK, so UnregisterClient does not use `mCheckInNodeId`?  Why are we setting it there, then?

Reply:
removed, thanks